src/screens/IndexScreen.js

```diff
@@ -29,20 +29,34 @@ import { usePosition, useTrackScreenViewAsync } from '../hooks';
 
 const { MATOMO_TRACKING } = consts;
 
+const TopFilter = {
```

Comment:
like `INITIAL_TOP_FILTER` this could be `TOP_FILTER`, because this is used as holding constant? as well as `LIST` and `MAP`? and then this would be fit very well placed in our `consts.js` file, where we already import `MATOMO_TRACKING` from, right?

Reply:
I capitalized the names, but kept them in this file. they are specific to this screen and there is no need to extract them from here imo.